Mlem/App/Views/Pages/ImageViewer.swift

```diff
@@ -153,6 +151,87 @@ struct ImageViewer: View {
         .statusBarHidden(!isDismissing)
     }
     
+    func handleDragGesture(value: DragGesture.Value) {
+        guard !isZoomed else {
+            return
+        }
+        
+        if dragIsScrub == nil {
+            dragIsScrub = abs(value.velocity.height) < abs(value.velocity.width)
+        }
+        
+        guard let dragIsScrub else {
+            assertionFailure("DragGesture updated but dragIsScrub is nil")
+            return
+        }
```

Comment:
Could we do something like:

```suggestion
        let dragIsScrub = self.dragIsScrub ?? abs(value.velocity.height) < abs(value.velocity.width)
        self.dragIsScrub = dragIsScrub
```

Reply:
That is nicer--I'm keeping the assertion, though, since gesture-heavy views are race condition hell